openpoiservice/server/db_import/objects.py

```diff
@@ -26,3 +29,31 @@ def __init__(self, uuid, osmid, key, value):
         self.osmid = int(osmid)
         self.key = key
         self.value = value
+
+
+class AddressObject(object):
+
+    def __init__(self, lat_lng):
+        self.lat_lng = lat_lng[::-1]
+
+    def address_request(self):
+
+        # try:
+        geocoder_settings = list(ops_settings['geocoder'].items())[0]
+        geolocator = get_geocoder_for_service(geocoder_settings[0])
```

Comment:
what happens if list(ops_settings['geocoder'].items())[0] is something get_geocoder_for_service doesn't expose?